src/utils/project.ts

```diff
@@ -0,0 +1,10 @@
+import { NotifyClientTypes } from '@walletconnect/notify-client'
+
+import { INotifyApp } from '@/utils/types'
+
+export function isUnVerified(app: NotifyClientTypes.NotifySubscription, projects: INotifyApp[]) {
+  const project = projects.find(project => project.url.includes(app.metadata.appDomain))
+  const isVerified = project && project?.isVerified !== true
+
+  return isVerified
```

Comment:
You could make this a one liner using `some`
```suggestion
  return projects.some(project => project.url.includes(app.metadata.appDomain) && project.isVerified !== true)
```